updating/preparing_for_updates/updating-cluster-prepare.adoc

```diff
@@ -12,7 +12,12 @@ WARNING: This assembly has been moved into a subdirectory for 4.14+. Changes to
 To do: Remove this comment once 4.13 docs are EOL.
 ////
 
-Learn more about administrative tasks that cluster admins must perform to successfully initialize an update, as well as optional guidelines for ensuring a successful update.
+Learn more about administrative tasks that cluster admins must perform to successfully initialize an update and optional guidelines for ensuring a successful update.
```

Comment:
Not really in the scope of your changes, but I find tension between the assembly title and this intro. It's the flow here is really "prepare" rather than "learn," I think?

Reply:
:heavy_check_mark: